app/models/in_person_enrollment.rb

```diff
@@ -12,8 +12,37 @@ class InPersonEnrollment < ApplicationRecord
 
   validate :profile_belongs_to_user
 
+  before_save(:on_status_updated, if: :will_save_change_to_status?)
```

Comment:
I like this way better, because you don't have to remember to manually update the timestamp every time you update the status (which we currently do in a few different places)